newton/_src/viewer/viewer_viser.py

```
@@ -64,6 +67,44 @@ def _to_numpy(x) -> np.ndarray | None:
+        return normalize_texture(
+            load_texture(texture),
+            flip_vertical=False,
+            require_channels=True,
+            scale_unit_range=True,
+        )
+
+    @staticmethod
+    def _build_trimesh_mesh(points: np.ndarray, indices: np.ndarray, uvs: np.ndarray, texture: np.ndarray):
+        """Create a trimesh object with texture visuals (if trimesh is available)."""
+        try:
+            import trimesh  # noqa: PLC0415
```

Ruff (0.14.14)

[warning] Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)